master/pkg/model/maintenance_message.go

```diff
@@ -0,0 +1,38 @@
+package model
+
+import (
+	"database/sql"
+	"time"
+
+	"google.golang.org/protobuf/types/known/timestamppb"
+
+	"github.com/determined-ai/determined/proto/pkg/apiv1"
+)
+
+// ClusterMessage represents a server status from the `cluster_messages` table.
+type ClusterMessage struct {
+	CreatedBy   int
```

Comment:
non-blocking: I usually embed the bun able in the struct, was there a reason not to?

Reply:
[Bun has defaults](https://bun.uptrace.dev/guide/models.html) that should cover this name.

Reply:
yeah I generally like the clarity, but I made it non-blocking because I feel like the comment fulfills that purpose. Either way, my approval still stands!